### ISSUE TITLE
Crete service accounts 

### ISSUE THREAD

**Issue by grahamcrowell** [VCHDecisionSupport/Azure_IaaS#65]
Match on prem with get-aduser 

- Create AD group? 
- creates aduser
- add to local admin group on each box

**Comment by grahamcrowell** [comment]
Depends on #43

**Comment by grahamcrowell** [comment]
see New-User.ps1

**Comment by grahamcrowell** [comment]
duplicate of #58 and #41
